### PR TITLE
first pass cleanup: mainly remove duplication of FDK docs

### PR DESCRIPTION
**Pull request by carimura**

### CODE REVIEW

**Comment by carimura** [comment]
ok @rikgibson @shaunsmith think this actually helps your cause now... after we get this one through we can address the de-duping FDK strategy.

**Comment by carimura** [comment]
@shaunsmith (or anyone) I recommend you review this PR through here: https://github.com/fnproject/docs/pull/45/files. It's quite simple.If you view all the commits individually you'll get the noise from taking the FDK stuff out then adding back in and it'll look like new files (per your comments thus far.)

File: README.md [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
-If you are new to Fn, check out this [introduction](fn/general/introduction.md). 
+## Welcome
+Fn is an event-driven, open source, [Functions-as-a-Service (FaaS)](https://github.com/fnproject/docs/blob/master/fn/general/introduction.md) compute platform that you can run anywhere. Some of its key features:
+
+* Open Source
+* Native Docker: use any Docker container as your Function
+* Supports all languages
+* Run anywhere
+  * Public, private and hybrid cloud
+  * Import Lambda functions and run them anywhere
```
Comment (denismakogon) [inline README.md]: this has to be tested, I'd rather ensured if it really works and then posts it.

File: README.md [hunk 2]
```diff
@@ -35,30 +52,44 @@ If you are operating Fn, this section is for you.
...
 
-## [CLI Documentation](cli/README.md)
-* [CLI Reference](cli/README.md#fn-command-reference)
-* [CLI How-to Guides](cli/README.md#fn-cli-how-to-guides)
+* [Ask your question on StackOverflow](https://stackoverflow.com/questions/tagged/fn) and tag it with `fn`
+* Join our [Slack Community](http://slack.fnproject.io)
+* Join us at one of our [Fn Events](http://events.fnproject.io) or even speak at one!
+* Coming in Q1'19: Regularly scheduled planning meetings for contributing to the Fn Project
+* For a full presentation with lots of content you can use in your own presentations, see [The Fn Project Presentation Master](http://deck.fnproject.io)
+* Check out our sub-projects: [Flow](https://github.com/fnproject/flow), [UI](https://github.com/fnproject/ui), [FnLB](https://github.com/fnproject/lb)
```
Comment (denismakogon) [inline README.md]: can you please add helm chart here as well?

File: README.md [hunk 3]
```diff
@@ -35,30 +52,44 @@ If you are operating Fn, this section is for you.
...
 
-## [FDKs](fdks/README.md)
 
-## [Examples](examples/README.md)
+### Stay Informed
 
-## [Tutorials](https://fnproject.io/tutorials)
+* [Blog](https://medium.com/fnproject)
+* [Twitter](https://twitter.com/fnproject)
+* [YouTube](https://www.youtube.com/channel/UCo3fJqEGRx9PW_ODXk3b1nw)
```
Comment (denismakogon) [inline README.md]: As David mentioned before, if we're not posting videos, then what's the point for promoting the channel?

File: fdks/README.md [hunk 4]
```diff
@@ -1,9 +1,7 @@
...
-### [fdk-go](fdk-go/README.md)
-### [fdk-java](fdk-java/README.md)
-### [fdk-node](fdk-node/README.md)
-### [fdk-python](fdk-python/README.md)
-### [fdk-ruby](fdk-ruby/README.md)
+FDKs are language kits designed to help make working with Fn easier in each language.
 
-For information on the [Fn container contract and the FDK click here.](../fn/develop/fn-format.md)
\ No newline at end of file
+Select one of the FDK folders above to view documentation on that specific FDK.
```
Comment (rdallman) [inline fdks/README.md]: the folders above have been deleted now

File: README.md (hunk 2 above)
Comment (rdallman) [reply README.md]: it's in the 'Learn more' section above, I think it fits better there too

Comment (rdallman) [reply README.md]: delete lb here please, and the UI doesn't even work, we should probably keep it hidden like boo radley

File: README.md [hunk 5]
```diff
@@ -35,30 +52,44 @@ If you are operating Fn, this section is for you.
...
 If you are working on the Fn Project, want to work on it or are creating extensions, this section is for you.
 
 * [Writing Extensions](fn/develop/extensions.md)
 * [Using Annotations](fn/develop/annotations.md)
+* Join our [Slack Community](http://slack.fnproject.io)
+* See our new [Community Page](https://github.com/fnproject/docs/blob/master/COMMUNITY.md)
+* Learn how to [contribute](CONTRIBUTING.md)
+* Find [issues](https://github.com/fnproject/fn/issues) and become a contributor
+* Join us at one of our [Fn Events](http://events.fnproject.io) or even speak at one!
+* Coming in Q1'19: Regularly scheduled planning meetings for contributing to the Fn Project
```
Comment (rdallman) [inline README.md]: this is just making stale docs, can we remove line 77 until we have meetings we can link to?

File: README.md [hunk 6]
```diff
@@ -35,30 +52,44 @@ If you are operating Fn, this section is for you.
...
+
+## Other Resources
 
-## [CLI Documentation](cli/README.md)
-* [CLI Reference](cli/README.md#fn-command-reference)
-* [CLI How-to Guides](cli/README.md#fn-cli-how-to-guides)
+* [Ask your question on StackOverflow](https://stackoverflow.com/questions/tagged/fn) and tag it with `fn`
+* Join our [Slack Community](http://slack.fnproject.io)
+* Join us at one of our [Fn Events](http://events.fnproject.io) or even speak at one!
+* Coming in Q1'19: Regularly scheduled planning meetings for contributing to the Fn Project
```
Comment (rdallman) [inline README.md]: dupe line of 77

Comment (rdallman) [reply README.md]: most of these are dupes actually, can we filter these?

File: README.md (hunk 1 above)
Comment (rdallman) [reply README.md]: yea i'm about 100% sure these are broken (they don't support http-stream format)

Comment (rdallman) [reply README.md]: we should use node-fdk, and fix them up, but anyway, yea, agree... broken, shouldn't promote

File: README.md (hunk 3 above)
Comment (carimura) [reply README.md]: let's post some videos then ;)

File: README.md (hunk 2 above)
Comment (carimura) [reply README.md]: lb, done. pr coming.

File: README.md (hunk 1 above)
Comment (carimura) [reply README.md]: removed.

File: fdks/README.md (hunk 4 above)
Comment (carimura) [reply fdks/README.md]: well, I took them out, then added them back due to debate on where FDK docs go.

File: README.md (hunk 5 above)
Comment (carimura) [reply README.md]: such lack of faith!! updating...

Comment (rdallman) [reply README.md]: without dates it would be fine :) it's just leaving stale docs to set dates here though...

**Comment by carimura** [comment]
ok @rikgibson @michael-w-williams got some reviews on this. As noted above I added back the FDK docs to simplify this PR. Can one of you approve so we can move forward? Then I'll go through again to ensure all the FDK stuff is up-to-date and we can discuss how to remove duplication.

File: README.md [hunk 7]
```diff
@@ -74,17 +72,15 @@ If you are working on the Fn Project, want to work on it or are creating extensi
 * Learn how to [contribute](CONTRIBUTING.md)
```
Comment (michael-w-williams) [inline README.md]: Should link to: https://github.com/fnproject/fn/blob/master/CONTRIBUTING.md. Otherwise it looks good.

Comment (carimura) [reply README.md]: good call -- going to move this over to docs repo right now actually... stand by for next PR.

**Comment by carimura** [comment]
thanks all.

**Pull request merged**